test/CMakeLists.txt

```diff
@@ -297,3 +297,8 @@ target_link_libraries(savecontext-bench benchmark::benchmark benchmark::benchmar
 target_include_directories(savecontext-bench PRIVATE ${DDPROF_INCLUDE_LIST})
 # clang-tidy does like google-benchmark macros
 disable_clangtidy(savecontext-bench)
+
+
+# add_executable(ringbuffer-ut ringbuffer-ut.cc ../src/ipc.cc)
```

Comment:
[minor] Some test code remains around here

Reply:
I am working on adding tests / benchmarks, this is part of it ;)